modules/ocf_desktop/manifests/packages.pp

```diff
@@ -84,11 +84,14 @@
       recommends => false;
   }
 
-  # Install libegl1-mesa-dev from stretch-backports so that libgtk-3-dev can be
-  # properly installed
   ocf::repackage {
+    # Install libegl1-mesa-dev from stretch-backports so that libgtk-3-dev can be
+    # properly installed
     'libegl1-mesa-dev':
       backport_on => 'stretch',
       before      => Package['libgtk-3-dev'];
+    # tilix is only available in backports
```

Comment:
Nit: would put empty line before here

Reply:
For the rest of the file, comments are not separated by newlines.